packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -97,6 +99,7 @@ class Dismissible extends StatefulWidget {
   }) : assert(key != null),
        assert(secondaryBackground == null || background != null),
        assert(dragStartBehavior != null),
+       assert(child != null || direction != null),
```

Comment:
Can we add more to the documentation, or an assertion message here that explains this?